src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt

```diff
@@ -0,0 +1,35 @@
+package at.hannibal2.skyhanni.features.event.jerry
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.EntityMaxHealthUpdateEvent
+import at.hannibal2.skyhanni.events.withAlpha
+import at.hannibal2.skyhanni.mixins.hooks.RenderLivingEntityHelper
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import net.minecraft.entity.Entity
+import net.minecraft.entity.player.EntityPlayer
+import net.minecraftforge.event.entity.EntityJoinWorldEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class HighlightInquisitors {
+    private val config get() = SkyHanniMod.feature.event.diana
+
+    @SubscribeEvent
+    fun onEntityHealthUpdate(event: EntityMaxHealthUpdateEvent) {
```

Comment:
question, why do you need this event. EntityJoinWorld should be fine to get the entity.

Reply:
If you spawn the entity it joins before receiving a name sometimes, so it gets caught by this. If other players spawn the entity, and you just get close enough to load it, the join world event catches it. Overall i've been pondering a rework for how mobs are detected.

Reply:
@brandonwamboldt  There is already a big open pr that deals with mob spawn and despawn detection

Also this pr could be the last feature pr before 0.22

Reply:
Player Entitys are should always spawn with the name only other mobs are a problem. And for the mob detection part, I have my [PR#712](https://github.com/hannibal002/SkyHanni/pull/712) for that.

Reply:
Removed the max health update code 👍